live-site/components/hacking-remaning/HackingRemaining.styles.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/lib/typography';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledHackingRemainingContainer = styled.div`
+  z-index: 21;
+  background-color: ${colors.BUTTON_DARK_GREEN};
+  position: fixed;
+  right: 0;
+  top: 4em;
+
+  width: 133px;
+  height: 139px;
+
+  text-align: center;
+
+  border-radius: 1em 0em 0em 1em;
+  border: 0.3em solid ${colors.BUTTON_DARK_GREEN};
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25))
+    drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
```

Comment:
Add padding-top: 1em; 

so that the hacking remaining text isn't too close to the edge